src/Symfony/Component/DependencyInjection/Tests/Loader/XmlFileLoaderTest.php

```diff
@@ -207,6 +208,20 @@ public function testParsesTags()
         }
     }
 
+    public function testParseTagsWithoutName()
+    {
+        try {
+            $container = new ContainerBuilder();
+            $loader = new XmlFileLoader($container, new FileLocator(self::$fixturesPath.'/xml'));
```

Comment:
We cannot can use it here as we are interested in the message of the wrapped exception.